apollo-router/src/plugins/record_replay/record.rs

```diff
@@ -250,43 +250,35 @@ impl Plugin for Record {
                     let subgraph_name = subgraph_name.clone();
                     async move {
                         let res: subgraph::ServiceResult = future.await;
+                        let res = res?;
```

Comment:
The `match` below just returns the error, which is what `?` syntax already achieves.